modules/update-restricted.adoc

```diff
@@ -23,6 +23,11 @@ If you have a local OpenShift Update Service, you can update by using the connec
 * Install the OpenShift CLI (`oc`).
 * Pause all `MachineHealthCheck` resources.
 
+[NOTE]
+====
+The release image signature config map allows the Cluster Version Operator (CVO) to verify that the release images have not been modified by comparing the expected and actual image signatures.
+====
```

Comment:
This note looks good to me, I'd just suggest adding it under line 21 so that it is closer to the content that it adds context to.